tests/configs/basic_app/k8s_cronjobs.yml

```diff
@@ -14,6 +14,8 @@ spec:
   schedule: "*/1 * * * *"
   startingDeadlineSeconds: 10
   concurrencyPolicy: "Allow"
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 1
```

Comment:
Ref the other comment. This should not be present in this definition. Instead we should be relying on `k8s` default value.

Reply:
https://github.com/EDITD/kubetools/pull/166/commits/b60c09557b9a04a5ba8f7c486d8b8c36d424adb0